projects/packages/forms/src/blocks/contact-form/components/jetpack-field-width.js

```diff
@@ -11,21 +15,25 @@ export default function JetpackFieldWidth( { setAttributes, width } ) {
 			className="jetpack-field-label__width"
 			__nextHasNoMarginBottom={ true }
 		>
-			<BaseControl.VisualLabel>{ __( 'Field Width', 'jetpack-forms' ) }</BaseControl.VisualLabel>
-			<ButtonGroup aria-label={ __( 'Field Width', 'jetpack-forms' ) }>
+			<ToggleGroupControl
+				__next40pxDefaultSize
+				__nextHasNoMarginBottom
+				aria-label={ __( 'Field Width', 'jetpack-forms' ) }
```

Comment:
Would it make sense to just use "Width" here instead of `Field Width`. In the search block that is what is being used.

Reply:
Yep makes sense